installer/knative/description.yaml

```diff
@@ -12,7 +12,7 @@ install:
     namespace: knative-serving
     waitfor:
       - namespace: knative-serving
-        selector: all
+        selector: serving.knative.dev/release=v0.24.0
```

Comment:
Would be awesome if you found a way to make it version independent 🙂

Reply:
yeah, unfortunately this is the only label shared between all pods... and I thought: the version is already written on top of that description file anyway, so it's not such a big deal

Reply:
oh, wait, maybe I found something...

Reply:
Hm, no.
```kubectl get pods -n knative-serving -l "app in (activator,webhook,domain-mapping,controller,autoscaler,domainmapping-webhook,net-istio-webhook,net-istio-controller)"```
works with `kubectl get`, but the same syntax cannot be applied to `kubectl wait` command we're using when checking the status.